aws/table_aws_ec2_transit_gateway_route.go

```diff
@@ -0,0 +1,140 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEc2TransitGatewayRoute(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_transit_gateway_route",
+		Description: "AWS EC2 Transit Gateway Route",
+		List: &plugin.ListConfig{
+			ParentHydrate: listEc2TransitGatewayRouteTable,
+			Hydrate:       listEc2TransitGatewayRoute,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "transit_gateway_route_table_id",
+				Description: "The ID of the transit gateway route table.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "destination_cidr_block",
+				Description: "The CIDR block used for destination matches.",
+				Type:        proto.ColumnType_STRING,
```

Comment:
seems like the property stores `CIDR` range. If so, use `proto.ColumnType_CIDR` instead.